cypress/e2e/michael.cy.ts

```diff
@@ -17,5 +17,28 @@ describe('obstacle practice', () => {
         cy.visit('https://obstaclecourse.tricentis.com/Obstacles/60469')
         cy.get('#toscabot').dragTo('#to')
         cy.contains('Good job!')
-    })
+    });
+
+    it('41038', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/41038')
+        cy.contains('Click into my right half').click('right', { force: true })
```

Comment:
Do not understand why use a force option here. It asks you to click a coordinate instead of an element.

https://docs.cypress.io/api/commands/click#Specify-a-corner-of-the-element-to-click